src/main/java/ch/njol/skript/expressions/ExprLevelProgress.java

```diff
@@ -36,7 +36,7 @@
  * @author Peter Güttinger
  */
 @Name("Level Progress")
-@Description({"The progress a player has made until the next level. Remember that this value is between 0 and 1, not 0 and 100!",
+@Description({"The progress a player has made until he reaches the next level. Remember that this value is between 0 and 1, not 0 and 100!",

```

Comment:
**Not needed** and actually it should be "they reaches" as i know
```suggestion
@Description({"The progress a player has made until the next level. Remember that this value is between 0 and 1, not 0 and 100!",
```

Reply:
Already there is one comment for that :)